ManageCoursesUi.Tests/ViewModelHelperTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+using GovUk.Education.ManageCourses.ApiClient;
+using GovUk.Education.ManageCourses.Ui.Helpers;
+using GovUk.Education.ManageCourses.Ui.ViewModels;
+using NUnit.Framework;
+
+namespace ManageCoursesUi.Tests
+{
+    [TestFixture]
+    class ViewModelHelperTests
+    {
+        [Test]
+        [TestCase("", "f", "ss", "QTS full time with salary")]
+        [TestCase("", "F", "SS", "QTS full time with salary")]
+        [TestCase(null, "f", "ss", "QTS full time with salary")]
+        [TestCase(null, "F", "SS", "QTS full time with salary")]
+        [TestCase("pg", "f", "ss", "PGCE with QTS full time with salary")]
+        [TestCase("PG", "F", "SS", "PGCE with QTS full time with salary")]
+        [TestCase("pg", "p", "ss", "PGCE with QTS part time with salary")]
+        [TestCase("PG", "P", "SS", "PGCE with QTS part time with salary")]
+        [TestCase("", "f", "sd", "QTS full time")]
+        [TestCase("", "F", "SD", "QTS full time")]
+        [TestCase("", "", "SD", "QTS part time")]
+        [TestCase("", "", "", "QTS part time")]
+        [TestCase(null, "f", "sd", "QTS full time")]
+        [TestCase(null, "F", "SD", "QTS full time")]
+        [TestCase("pg", "f", "sd", "PGCE with QTS full time")]
+        [TestCase("PG", "F", "SD", "PGCE with QTS full time")]
+        [TestCase("pg", "p", "sd", "PGCE with QTS part time")]
+        [TestCase("PG", "P", "SD", "PGCE with QTS part time")]        
+        public void TestGetCourseVariantType(string profpostFlag, string studyMode, string programType, string expectedResult)
+        {
+            var course = new Course
+            {
+                ProfpostFlag = profpostFlag,
+                StudyMode = studyMode,
+                ProgramType = programType
+            };
+            var result = course.GetCourseVariantType();
+            Assert.IsTrue(result == expectedResult);
```

Comment:
Can we stop inline it like that it makes refactoring a pain
Use 
Assert.AreEqual(actual, expected)

Reply:
Also I'd like to see us use fluent assertions everywhere.

Reply:
Installed fluid assertions and now using them in all helper tests